src/main/business/MetaQuestions/MetaQuestionController.js

```diff
@@ -117,6 +117,12 @@ class MetaQuestionController{
         const dalMQs = await this.#metaQuestionRepo.getMetaQuestionsForAppendix(data.appendixTag);
         return dalMQs.map(dalMQ => new MetaQuestion(dalMQ));
     }
+
+    async getMetaQuestionsForExam(data){
+        validateParameters(data, {examId: PRIMITIVE_TYPES.NUMBER});
+        const dalMQs = await this.#metaQuestionRepo.getMetaQuestionsForExam(data.examId);
```

Comment:
`getMetaQuestionsForExam`
Didn't see you added it to the MetaQuestionRepository